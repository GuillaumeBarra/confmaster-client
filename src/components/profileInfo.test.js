import React from 'react'
import {render} from '@testing-library/react'
import ProfileInfo from './profileInfo'
 
 it('should take a snapshot', () => {
    const { asFragment } = render(<ProfileInfo />)
    
    expect(asFragment(<ProfileInfo />)).toMatchSnapshot()
   })