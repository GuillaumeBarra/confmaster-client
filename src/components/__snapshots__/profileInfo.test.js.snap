// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    style="margin-left: 20px;"
  >
    <div
      style="font-size: 20px; font-weight: 600; padding-top: 20px; padding-left: 20px;"
    />
    <div
      style="font-size: 18px; font-style: italic; padding-left: 20px;"
    />
    <div
      style="font-size: 18px; font-weight: 600; padding-left: 20px; padding-top: 20px;"
    >
      Current Conference:
    </div>
    <div
      style="font-size: 18px; font-style: italic; padding-left: 20px;"
    >
      , 
    </div>
    <div
      style="font-size: 18px; font-weight: 600; padding-left: 20px; padding-top: 20px;"
    >
      Previous Conferences:
    </div>
    <div
      style="font-size: 18px; font-style: italic; padding-left: 20px;"
    >
      , 
    </div>
  </div>
</DocumentFragment>
`;
